Commands/MasteryCommands.cs

```diff
@@ -90,6 +104,24 @@ private static string GetMasteryDataStringForType(WeaponMasterySystem.MasteryTyp
             // return $"{name}: <color={Output.White}>{mastery:F2}%</color> ({string.Join(",", statData)}) Effectiveness: {effectiveness * 100}%, Growth: {growth * 100}%";
         }
 
+        private static void PrintMasteryDetails(ChatCommandContext ctx)
+        {
+            var masteryType = WeaponMasterySystem.WeaponToMasteryType(WeaponMasterySystem.GetWeaponType(ctx.Event.SenderCharacterEntity, out var weaponEntity));
+            var weaponMasteryPrintableData = WeaponMasterySystem.GetWeaponMasteryPrintableData(ctx.User.PlatformId, masteryType, weaponEntity);
+            var spellMasteryPrintableData = WeaponMasterySystem.GetSpellMasteryPrintableData(ctx.User.PlatformId);
+            var language = L10N.GetUserLanguage(ctx.User.PlatformId);
+
+            foreach (var line in weaponMasteryPrintableData)
+            {
+                ctx.Reply(line.Build(language));
```

Comment:
If really want to have a dedicated print for mastery, you should use `Output.ChatReply(ctx, weaponMasteryPrintableData.ToArray());` as that should format it better without adding in all the extra `[SYSTEM]` parts of the reply.

Reply:
Maybe we can keep it until we improve the player info command. However, printing too much info makes it very difficult to read. I moslty made this command for debug because is difficult to see the buffs within the game.